recipe/meta.yaml

```
@@ -62,6 +59,7 @@ test:
     - ipython -h
     - ipython3 -h
     - export MIGRATING=false  # [unix]
+    - pytest tests -v
```

```suggestion
    - pytest tests -v ./tests
```
Probably makes no difference here but is good practice to point directly to the test suite. pytest's module discovery can sometimes do some weird stuff if the source and tests are both present and it's allowed to traverse into both directories.